src/postgres/catalog.js

```diff
@@ -120,10 +120,11 @@ export class Schema {
  * @member {ForeignKey[]} reverseForeignKeys
  */
 export class Table {
-  constructor ({ schema, name, description }) {
+  constructor ({ schema, name, description, kind }) {
     this.schema = schema
     this.name = name
     this.description = description
+    this.kind = kind
```

Comment:
Could you create some constants for `kind`? So for example:

```js
export const KIND_ORDINARY_TABLE = 'r'
export const KIND_VIEW = 'v'
…
```

This could help contributors in the future. If you could also add `kind` to the JSDoc above that would be great 😊👍